src/editor/main.js

```diff
@@ -210,3 +213,14 @@ document.querySelectorAll('.toggle-layers').forEach((element) => {
     document.body.classList.toggle('open')
   );
 });
+
+document.querySelector('.sizes').addEventListener('change', (evt) => {
+  const checkbox = /** @type {HTMLInputElement} */ (evt.target);
+  let size = /** @type {Number} */ parseInt(checkbox.name.substring(1), 10);
+
+  if (checkbox.checked && size != 0) {
+    exportSizeArray.push(size)
```

Comment:
I think using `new FormData(...).get('size')` would be more succinct than managing the array.

Reply:
Added Form and using it in export.